src/sentry/dynamic_sampling/tasks/recalibrate_orgs.py

```diff
@@ -88,14 +91,46 @@ def recalibrate_org(org_volume: OrganizationDataVolume, context: TaskContext) ->
 
         assert org_volume.indexed is not None
 
-        target_sample_rate = get_sliding_window_org_sample_rate(org_volume.org_id)
-        log_sample_rate_source(
-            org_volume.org_id, None, "recalibrate_orgs", "sliding_window_org", target_sample_rate
+        try:
+            # We need the organization object for the feature flag.
+            organization = Organization.objects.get_from_cache(id=org_volume.org_id)
```

Comment:
Will have to observe the org loading and the performance impact on the task. Shouldn't be a big problem considering we use this function across the other cron jobs.

Reply:
It's already instrumented, Radu built a sophisticated system to keep track of tasks execution time after the last incident, so we are good.